README.md

```diff
@@ -16,64 +15,71 @@ Listen to the corresponding hourly music from the main Animal Crossing titles:
 
-To install the downloaded project, do the following:
-1. Go to the [Chrome Extensions Page](chrome://extensions/), which can be found under `Chrome > Settings > Extensions` 
-2. Enable the `Developer Mode` switch
-3. Click on `Load Unpacked`
-4. Choose the unzipped project directory
+The extension no longer contains music files, and the source code does not depend on resources hosted elsewhere.
 
+Thanks to this change, the size of the extension has shrink from `436MB` to only `4,1MB`. That's a 99,99% decrease in size!
 
-### Resources
-Currently, music resources are packed within the extension. As they take up a lot of space, they have been purposedly left out of the repository.
+Downloading the music files and including them into the project directory **is no longer needed** in order to install/use the extension. 
 
-You can find the music resources in the following link:
+You can still find the used music resources in the following link:
 - Google Drive: [Animal Crossing Music Extension Revived](https://drive.google.com/open?id=1QjG8QjPt_IKgzZRzmnoH8KTGwTsWSLsR)
 
-In order to run this project locally, you will need to download the above directory and extract it under `./sounds`.
+### Branch conventions
+- The `master` branch contains the latest published version of the extension. This is the stable branch.
+- The `develop` branch contains the latest fully-integrated features of the extension. This is where PRs should be merged into. 
 
-If, for some reason, you need access to the legacy music and structure, you can find them in the following link:
-- Google Drive: [Animal Crossing Music Extension](https://drive.google.com/drive/folders/0B79uF1ZqAHtbN0l4eFR1NU9CMGc)
+## Support
+If you have any questions or doubts, feel free to jump into the project's [discord channel](https://discordapp.com/invite/4FMrEF8).
 
 ## Contribute
-Contributions to the project are greatly appreciated!
-
-If you have a feature request, a proposal for improvement, want to report a bug or just to ask a question, feel free to open a public [issue](https://github.com/PikaDude/Animal-Crossing-Music-Extension/issues) on the project repository.
+Contributions to the project are **greatly** appreciated! Here is how you can help to make this project even better:
+### Issues
+If you want to report a bug, or have a feature request/proposal for improvement, feel free to open a public [issue](https://github.com/PikaDude/Animal-Crossing-Music-Extension/issues) on the project repository.
 
-[PRs](https://github.com/PikaDude/Animal-Crossing-Music-Extension/pulls) are equally welcomed. Just make sure to mention a relevant issue in its description!
+### PR Submission
+[PRs](https://github.com/PikaDude/Animal-Crossing-Music-Extension/pulls) are equally welcomed. Just make sure to:
 
+* Any changes you introduce **do not** break the project in any way
+* You have mentioned any relevant issue in its description
+* You choose `develop` as base branch
```

Comment:
I came up with these PR requirements on the spot, do you agree?

Reply:
Trying to ask people not to submit PRs that break the project is kinda vague and sometimes hard for them to confirm and verify, it's best that we do these tests ourselves and don't bother mentioning it.

Reply:
Then I think we should just point out that they should submit them`develop`